crates/nu-command/src/commands/run_external.rs

```diff
@@ -86,12 +84,9 @@ impl WholeStreamCommand for RunExternalCommand {
                 ctrl_c: args.ctrl_c.clone(),
                 configs: args.configs.clone(),
                 current_errors: Arc::new(Mutex::new(vec![])),
-                windows_drives_previous_cwd: Arc::new(Mutex::new(std::collections::HashMap::new())),
```

Comment:
windows_drives_previous_cwd is moved from ctx into shell_manager.

This variable is only used for cd'ing and is therefore closer to shellmanager